app/client/cli/helpers/setup.go

```diff
@@ -4,26 +4,37 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/pokt-network/pocket/app/client/cli/flags"
 	"github.com/pokt-network/pocket/logger"
 	"github.com/pokt-network/pocket/p2p"
 	rpcCHP "github.com/pokt-network/pocket/p2p/providers/current_height_provider/rpc"
 	rpcPSP "github.com/pokt-network/pocket/p2p/providers/peerstore_provider/rpc"
 	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/configs"
 	"github.com/pokt-network/pocket/shared/modules"
 )
 
+// TODO_THIS_COMMIT: add godoc comment explaining what this **is** and **is not**
```

Comment:
```suggestion
// TODO_IN_THIS_COMMIT: add godoc comment explaining what this **is** and **is not**
```

OPTIONAL: Update `build/linters/blockers.go` so we capture other regexes as well

Reply:
Added a comment here to capture the meaning of the var